base-notebook/Dockerfile

```diff
@@ -2,9 +2,7 @@
 # Distributed under the terms of the Modified BSD License.
 
 # Ubuntu 20.04 (focal)
-# https://hub.docker.com/_/ubuntu/?tab=tags&name=focal
```

Comment:
This line is ok, it doesn't hardcode arch

Reply:
Sounds good, I'll restore it.